<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="structure">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>树模式</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>自由模式</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="nodeform">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>圆图</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>框图</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>结构形式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>结点形式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="makegragh">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>画图</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lead_out_path">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>631</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>D:\QGraphtest.jpg</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>保存路径</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>631</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>数据域</string>
    </property>
   </widget>
   <widget class="QComboBox" name="lineform">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>无向图</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>有向图</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>连接形式</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lead_in_path">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>520</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="lead_in">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>导入</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>导入路径</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
